lib/bulk_insert/worker.rb

```diff
@@ -107,6 +115,7 @@ def compose_insert_query
       if !rows.empty?
         sql << rows.join(",")
         sql << on_conflict_statement
+        sql << primary_key_return_statement
```

Comment:
I think there may be a use case for making returning primary keys optional so we only do it when we need to. It would at least give more confidence that this is not going to break an existing places we use bulk insert in case we rely on the return value somewhere.

Reply:
Another reason to make it optional is so that the return isn't _different_ depending on what backend you happen to be using. Inconsistency in interfaces should be opt in.

Reply:
Updated the primary_key_return_statement method to make it opt-in